### PR TITLE
Allow chrome.hid for Crypto Wallets extension

### PR DESCRIPTION
**Pull request by bbondy**
Resolves https://github.com/brave/brave-browser/issues/16305
Description of the fix in the issue.

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by onyb** [comment]
Probably a dumb question  -  where did you get the `21070F3D60711361C1210B870439BE49B5D995F4` identifier from? Does it change with every new build of ethereum-remote-client?

**Review by mkarolin — APPROVED** [review]
`patches` ++

**Review by onyb — APPROVED** [review]

**Review by fmarier — APPROVED** [review]
This enables these two extensions APIs:
- `hid`
- `usbDevices`

File: patches/extensions-common-api-_permission_features.json.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
+diff --git a/extensions/common/api/_permission_features.json b/extensions/common/api/_permission_features.json
+index c89ba3f84c35ed0dff9bb26fb581edbfbff6f90c..da57157cba93500b08e2a79e4c0a486c836805cc 100644
+--- a/extensions/common/api/_permission_features.json
++++ b/extensions/common/api/_permission_features.json
+@@ -328,6 +328,7 @@
+       "channel": "stable",
+       "extension_types": ["extension"],
+       "allowlist": [
++        "21070F3D60711361C1210B870439BE49B5D995F4",  // Ethereum Remote Client extension
```
Comment (bridiver) [inline patches/extensions-common-api-_permission_features.json.patch:9]: this should have gone in https://github.com/brave/brave-core/blob/master/common/extensions/api/_permission_features.json or better yet we should dump the existing patch in feature_compiler.py and do a chromium_src override for `AddCorePermissionFeatures` in the permission_features.h generated file

**Pull request merged**